entrypoint.sh

```diff
@@ -189,9 +188,11 @@ query_string="?preview_theme_id=${preview_id}&_fd=0&pb=0"
 min_score_performance="${LHCI_MIN_SCORE_PERFORMANCE:-0.6}"
 min_score_accessibility="${LHCI_MIN_SCORE_ACCESSIBILITY:-0.9}"
 
+
 cat <<- EOF > lighthouserc.yml
 ci:
   collect:
+    chromePath: "$(which chromium)"
```

Comment:
When `puppeteerScript` is given but no `chromePath`,
Lighthouse CI was shouting "Chrome not found".
Therefore, I set the path to chromium manually.